### ISSUE TITLE
wordlist should have an index

### ISSUE THREAD

**Issue by sebbASF** [inclusivenaming/org#78]
The wordlst page: https://inclusivenaming.org/language/word-list/ should have an index.

It only contains 3 entries so far, and it is already too big to see them all without scrolling.

**Comment by richsalz** [comment]
I would go further, and make each entry be its own web page.  That makes it easier for others to link to specific words in their own policies, etc.  I'd be willing to make a PR in the https://github.com/inclusivenaming/website/ repo if this was wanted.

**Comment by lex-joy** [comment]
If feasible, separate webpages would likely be better for accessibility
purposes -- easier on screen readers, lesser loading times, etc. I'd vote
for this approach.

**Comment by markcmiller86** [comment]
It looks like the site is manged with Hugo (static site gen). I have experience only with Jekyll. In Jekyll there is an abstraction for *collections* and then each word/recommendation would be a member of a *collection* and generation of the index or toc or whatever (using Liquid templating language) would be more or less free...no work as new words are added.

**Comment by sebbASF** [comment]
Yes, individual pages would be better, but please retain the existing page as an index.

Multiple pages would allow for longer discussions of alternatives for particular words.
